biz.aQute.repository/src/aQute/bnd/repository/maven/pom/provider/BndPomRepository.java

```diff
@@ -44,27 +59,32 @@
  */
 @BndPlugin(name = "PomRepository")
 public class BndPomRepository extends BaseRepository
-		implements Plugin, RegistryPlugin, RepositoryPlugin, Refreshable, Actionable {
-	static final String			MAVEN_REPO_LOCAL	= System.getProperty("maven.repo.local", "~/.m2/repository");
-
-	private boolean				inited;
-	private PomConfiguration	configuration;
-	private Registry			registry;
-	private String				name;
-	private Reporter			reporter			= new Slf4jReporter(BndPomRepository.class);
-	private InnerRepository	repoImpl;
-	private List<Revision>		revisions;
-	private BridgeRepository	bridge;
-	private List<URI>			pomFiles;
-	private String				query;
-	private String				queryUrl;
+		implements Plugin, RegistryPlugin, RepositoryPlugin, Refreshable, Actionable, Closeable {
+	private final static Logger		logger				= LoggerFactory.getLogger(BndPomRepository.class);
+	static final String				MAVEN_REPO_LOCAL	= System.getProperty("maven.repo.local", "~/.m2/repository");
+	static int						DEFAULT_POLL_TIME	= (int) TimeUnit.SECONDS.toMillis(5);
```

Comment:
This should be in seconds.

https://github.com/bndtools/bnd/blob/5d8500fd174eb7ec4228f383baba910d4d5573f8/biz.aQute.repository/src/aQute/bnd/repository/osgi/OSGiRepository.java#L53

And a default time of 5 seconds is way too short. Use 5min like in OSGiRepository.